client/errors.go

```diff
@@ -166,6 +166,7 @@ var (
 	encAuthRegex   = regexp.MustCompile(`(\s)(Encoded authorization failure message:)\s[A-Za-z0-9_-]+`)
 	userRegex      = regexp.MustCompile(`(\s)(is not authorized to perform: .+ on resource:\s)(user)\s.+`)
 	s3Regex        = regexp.MustCompile(`(\s)(S3(Key|Bucket))=(.+?)([,;\s])`)
+	snapshotRegex  = regexp.MustCompile(`(\s)(The snapshot 'snap-)(.+?)'( does not exist)`)
```

Comment:
I think maybe ` snapshot 'snap-` all of this can also be part of the regex as we can probably get similar errors of 
The instance i-adsa12321 does not exist.

Wdyt?

Reply:
Sure that makes sense. Let me update it

Reply:
Please let me know if https://github.com/cloudquery/cq-provider-aws/pull/821/commits/c7e4259e1f163eec85f0b4debcf283b76281cc13 is what you had in mind. I made the regex more generic, but still kept the resource type (so we don't group instance+snapshot)

Reply:
Sounds great, yeah we should group same resoruces